### PR TITLE
Basic UI with minimal CSS

### LINKED ISSUE #1: UI Without Function

### PR DESCRIPTION
**Pull request by dallen68**

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    text-align: center;
+}
+
+.promoCodeList {
+    text-align: center;
+    margin-top: 100px
+}
+.promoCodeList table{
+    display: inline;
+}
```
Comment (josefj1519) [inline src/main/webapp/style.css:24]: Is there a new line at the end of file? I can't tell with github diffs.

Comment (dallen68) [reply src/main/webapp/style.css:24]: Yes github only shows changes

**Review by josefj1519 — APPROVED** [review]
LGTM

**Comment by dallen68** [comment]
Screenshot: vacnfqbU3HS

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+                    <th>VideoURL</th>
+                    <th>PROMO CODE</th>
+                    <th>Rating</th>
+                    <th><button>Like</button></th>
+                    <th><button>Dislike</button></th>
+                </tr>
+
+                <tr>
+                    <th><a href="youtube.com">youtube.com</a></th>
+                    <th>XXXXX</th>
```
Comment (derrillhd) [inline src/main/webapp/index.html:29]: nit: add a todo as to what goes here or will go here.  like <!-- TODO(dantebot): Add header --!>

**Review by derrillhd — APPROVED** [review]

File: src/main/webapp/index.html (hunk 2 above)
Comment (dallen68) [reply src/main/webapp/index.html:29]: I'm confused as to what you mean. That is just an example promo code listing

Comment (derrillhd) [reply src/main/webapp/index.html:29]: ah okay.  never mind.  Probably next step is to define a stub servlet methods that returns just a constants like

return new Gson("stuff", "stuff", "stuff");

Then you can try updating the FE to call the backend and it should render "stuff", "stuff", "stuff"

**Pull request merged**